front/components/assistant/conversation/ConversationTitle.tsx

```diff
@@ -193,7 +194,7 @@ export function ConversationTitle({
                 />
               )}
             </div>
-            {isDevelopment() && (
+            {(isDevelopment() || isADustProdWorkspace(owner)) && (
```

Comment:
let's limit to development we want to keep the product == to what our users see :pray:

Reply:
counter point: easily debug in prod and we already do it for "force admin|builder|user".

Reply:
It's quite hidden behind a rare click. this one is visible at all times.
Happy to have it in the menu next to dev tools when you're on a conversation page